modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -124,6 +124,222 @@
 		"issueKey": "LPD-26157",
 		"to": "StructureStructureKeyComparator.getInstance(false)"
 	},
+	{
+		"from": "new CommerceAddressCreateDateComparator(boolean paramName)",
+		"issueKey": "LPD-26935",
+		"skipParametersValidation": true,
+		"to": "CommerceAddressCreateDateComparator.getInstance(param#0#)"
+	},
+	{
+		"from": "new CommerceAddressNameComparator(boolean paramName)",
+		"issueKey": "LPD-26935",
```

Comment:
@victorhcunha @titokenzo The change in this file introduces 2 ticket numbers, while the PR's ticket is LPD-28287. Is it typo?

Reply:
Yes! It was a typo, I resent fixing.